### PR TITLE
Add sentence about ensuring the infrastructure

### PR DESCRIPTION
**Pull request by Ainali**
This makes it clear that we will help the community making high quality contributions.

-----
[View rendered activities/codebase-stewardship/goals.md](https://github.com/publiccodenet/about/blob/ja-ensure-infrastructure/activities/codebase-stewardship/goals.md)

### CODE REVIEW

**Comment by ericherman** [comment]
I'm 100% on board with the content.

I feel that the sentence wording is a bit klunky, but I'd ask @ElenaFdR to weigh in on better wordsmithing here.

File: activities/codebase-stewardship/goals.md [hunk 1]
```diff
@@ -23,5 +23,6 @@ Prospective users can trust there will be packages when they need to update thei
 Future contributors can trust that there will be someone to receive their pull request after the original maintainers have moved on.
 
 We provide this sustainability in the first place by creating strong communities of users and developers that can take ownership of the product.
+We ensure that the infrastructure that enables all members of the community making high quality contributions are in place.
```
Comment (bvhme) [inline activities/codebase-stewardship/goals.md:26]: What about taking a more active role?

```suggestion
We provide infrastructure that enables all members of the community to contribute. For instance mailing lists and for continuous integration testing.
```

**Review by bvhme — APPROVED** [review]

**Review by bvhme — DISMISSED** [review]

File: activities/codebase-stewardship/goals.md [hunk 2]
```diff
@@ -23,5 +23,6 @@ Prospective users can trust there will be packages when they need to update thei
 Future contributors can trust that there will be someone to receive their pull request after the original maintainers have moved on.
 
 We provide this sustainability in the first place by creating strong communities of users and developers that can take ownership of the product.
+We provide infrastructure that enables all members of the community to contribute. For instance mailing lists and for continuous integration testing.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/goals.md:26]:
```suggestion
We also provide infrastructure that enables all members of the community to contribute, for example mailing lists and tools for continuous integration testing.
```

**Review by ElenaFdR — COMMENTED** [review]
Alternative wordsmithing suggestion if you want it!

File: activities/codebase-stewardship/goals.md (hunk 2 above)
Comment (Ainali) [reply activities/codebase-stewardship/goals.md:26]: Great suggestion, thanks!

**Review by ericherman — APPROVED** [review]

**Pull request merged**